command.go

```diff
@@ -359,6 +361,7 @@ func parseRemoveArgs(args []string) (Command, error) {
 func parseShellArgs(args []string) (Command, error) {
 	flag := pflag.NewFlagSet("shell", pflag.ContinueOnError)
 	flag.String("assume", "", "Role to assume")
+	flag.Bool("refresh", false, "Refresh session expiration")
```

Comment:
This makes it sound like we're specifically extending the expiration of the session, which isn't true at all. Instead, we're ignoring any existing session, generating a new session with new temporary credentials. The help text should reflect as much.